java/client/test/org/openqa/selenium/devtools/ChromeDevToolsProfilerTest.java

```diff
@@ -0,0 +1,112 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.devtools;
+
+import static org.openqa.selenium.devtools.profiler.Profiler.consoleProfileFinished;
+import static org.openqa.selenium.devtools.profiler.Profiler.consoleProfileStarted;
+import static org.openqa.selenium.devtools.profiler.Profiler.disable;
+import static org.openqa.selenium.devtools.profiler.Profiler.enable;
+import static org.openqa.selenium.devtools.profiler.Profiler.getBestEffortCoverage;
+import static org.openqa.selenium.devtools.profiler.Profiler.setSamplingInterval;
+import static org.openqa.selenium.devtools.profiler.Profiler.start;
+import static org.openqa.selenium.devtools.profiler.Profiler.startPreciseCoverage;
+import static org.openqa.selenium.devtools.profiler.Profiler.startTypeProfile;
+import static org.openqa.selenium.devtools.profiler.Profiler.stop;
+import static org.openqa.selenium.devtools.profiler.Profiler.stopTypeProfile;
+import static org.openqa.selenium.devtools.profiler.Profiler.takePreciseCoverage;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.devtools.profiler.model.Profile;
+import org.openqa.selenium.devtools.profiler.model.ProfileNode;
+import org.openqa.selenium.devtools.profiler.model.ScriptCoverage;
+
+import java.util.List;
+import java.util.Optional;
+
+
+public class ChromeDevToolsProfilerTest extends ChromeDevToolsTestBase {
+
+
+
+  @Test
+  public void aSimpleStartStopAndGetProfilerTest() {
+    devTools.send(enable());
+    devTools.send(start());
+    Profile profiler = devTools.send(stop());
+    validateProfile(profiler);
+    devTools.send(disable());
+
+  }
+
+  private void validateProfile(Profile profiler) {
+    Assert.assertNotNull(profiler);
```

Comment:
Typically junit tests use static imports for `Assert` (and the assertion methods from assertj)